frontend/src/theme/utils.ts

```diff
@@ -118,9 +118,6 @@ export const createBaseThemePrimitives = (
     mono800: colors.bodyText, // Selectbox text
     mono900: colors.bodyText, // Not used, but just in case.
     mono1000: colors.black,
-
-    rating200: "#FFE1A5",
```

Comment:
Did `rating200` and `rating400` got replaced with any other theming option in baseweb UI that we might need to adapt?

Reply:
They seem to be removed, in [color-tokens.ts](https://github.com/uber/baseweb/blob/master/src/themes/light-theme/color-tokens.ts) we can find variables:
```
  // Rating Palette,
  ratingInactiveFill: colors.gray100,
  ratingStroke: colors.gray300,
```
which seem to govern Stroke and InactiveFill, but there is no 1:1 relation between them